DBTools/scripts/osusub.py

```diff
@@ -1338,12 +1347,12 @@ def copyDatasetInfoToTempFile (f, tempdir):
                     os.syetem('./lxbatchSub.sh')
                 else:
                     os.symlink ("../" + os.environ["CMSSW_VERSION"] + ".tar.gz", os.environ["CMSSW_VERSION"] + ".tar.gz")
-                    cmd = "condor_submit condor.sub"
-                    if os.path.isfile (proxy):
-                        os.chmod (proxy, 0644)
+                    cmd = "LD_LIBRARY_PATH= condor_submit condor.sub"
```

Comment:
I don't understand the `LD_LIBRARY_PATH= ` part. Why would you export an environment variable instead of executing the submit command?

Reply:
This may be some quirk of my environment, though I have no idea where it's coming from. When I try to run any of the condor commands after doing `cmsenv`, I get the following error:
```
Traceback (most recent call last):
  File "/usr/local/bin/condor_q", line 6, in <module>
    import argparse
  File "/usr/lib/python2.7/argparse.py", line 85, in <module>
    import collections as _collections
  File "/usr/lib/python2.7/collections.py", line 8, in <module>
    from _collections import deque, defaultdict
ImportError: /usr/lib/python2.7/lib-dynload/_collectionsmodule.so: wrong ELF class: ELFCLASS32
```
It only goes away if I unset `LD_LIBRARY_PATH`.

This line isn't exporting `LD_LIBRARY_PATH`. It is running `condor_submit condor.sub` with `LD_LIBRARY_PATH` set to nothing. This is different than exporting because the change to the environment persists only for the execution of the command.

Like I said, this is probably just a quirk of my environment, but unsetting `LD_LIBRARY_PATH` when executing condor commands in `osusub.py` shouldn't hurt anything for anyone else.